src/app/models/node-utilization.model.ts

```diff
@@ -25,4 +28,42 @@ export class NodeUtilization {
       nodeNames: null | string[];
     }[],
   ) {}
+
+  // transform NodeUtilization to NodeUtilizationChartData for NodeUtilization bar chart
+  toNodeUtilizationChartData(){
+    const MAX_NODES_IN_DESCRIPTION = 15;
+    const backgroundColor = CHART_COLORS[0];
```

Comment:
I have a small question. Is it intended for backgroundColor to always be CHART_COLORS[0]? In the original implementation, the bar's background color would change. However, in this implementation, the background color remains constant, so there seems to be no need to declare an array CHART_COLORS with multiple colors.

Reply:
Hi @wusamzong,
You're right, no need to declare multiple color here.  Just changed CHART_COLORS(Array) to CHART_COLOR(constant).

Thanks for the review.